actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -200,7 +200,7 @@ def favicon_link_tag(source = "favicon.ico", options = {})
       end
 
       # Returns an HTML image tag for the +source+. The +source+ can be a full
-      # path or a file.
+      # path, a file or an ActiveStorage attachment.
```

Comment:
When referring to the library and not the Ruby constant, leave a space between the words “Active” and “Storage.”